src/sentry/static/sentry/app/components/sidebar/sidebarPanelItem.tsx

```diff
@@ -51,7 +51,7 @@ const SidebarPanelItemRoot = styled('div')`
   border-bottom: 1px solid ${p => p.theme.innerBorder};
   background: ${p => p.theme.background};
   font-size: ${p => p.theme.fontSizeMedium};
-  padding: ${space(4)};
+  padding: ${space(3)};
```

Comment:
Does this affect other sidebars?

Reply:
Nothing in onboarding and what's new shares the same view as the one when Sentry has problems